src/components/icons/AddIcon.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
```

Comment:
No, these are placeholder icons.  I am waiting until after our design system talk / sync to begin implementation of the actual icons. Good question though.

Reply:
It will be easy to get this mixed up now that I'm about to merge some work with real icons. Can you rename these components to be `PlaceholderAddIcon` etc?
I know that's annoying. It's just, I just made a component called `PlusIcon` and when this is merged, and you're looking for the icon to use it's not going to be obvious which one is correct. Then when someone is implementing final designs on this page, they can delete the whole `PlaceholderAddIcon` component.

Reply:
Good point.  I totally agree that it's better to be clear now and avoid any confusion leading to code debt later. :100: